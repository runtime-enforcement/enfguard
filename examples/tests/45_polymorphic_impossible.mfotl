ALWAYS (FORALL x. A(x) AND x = "1" IMPLIES B(x))