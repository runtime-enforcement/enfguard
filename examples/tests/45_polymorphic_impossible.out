The formula
 □[0s,∞) (∀x. A(x) ∧ (x = "1") → B(x))
is enforceable and types to
 □[0s,∞) ((∀x. (A(x) ∧ (x = "1") →:R (B(x) : Cau) : Cau)) : Cau) : Cau
